.github/workflows/build.yml

```diff
@@ -88,7 +89,7 @@ jobs:
           CGO_ENABLED: 0
         run: |
           mkdir dist out
-          VAULT_VERSION=${{ needs.get-product-version.outputs.product-version }} VAULT_COMMIT=${GITHUB_SHA} GO_TAGS="${{ env.GO_TAGS }}" make build
+          GO_TAGS="${{ env.GO_TAGS }}" VAULT_VERSION=${{ needs.get-product-version.outputs.product-version }} VAULT_REVISION="$(git rev-parse HEAD)" make build
```

Comment:
Is `rev-parse HEAD` definitely right? This will not get the head of the release branch, but will get the specific SHA tied to this commit?

Reply:
Yep, that's right! This is the full commit from the branch that is building the artifacts. In the test example below: 

```
elle: ~/Users/elle/Downloads $ ./vault --version
Vault v1.10.0-dev (9e2f21337b47d8ef59f057b6d192d6fde2c5fb19)
```

[this commit](https://github.com/hashicorp/vault/commit/9e2f21337b47d8ef59f057b6d192d6fde2c5fb19` shows my changes that triggered the builds. When this is merged into a release branch, the artifacts built at that time will show the SHA from the merge commit. At release time, the artifacts will show the SHA from the final commit to the release branch. I think that's what we want

Reply:
This does that, yes!

Reply:
Today (before the staging releases tmr), I'll confirm the SHA's with you